editor-components/content/content13/content13.tsx

```diff
@@ -1,188 +1,127 @@
-              value:
-                "A dedicated server refers to a type of web hosting service where a single server is allocated exclusively to a single client or website. In other words, the entire physical server is dedicated to serving the needs of that client or website alone.",
-            },
-            {
-              type: "image",
-              key: "image",
-              displayer: "Image",
-              value: serverimg3,
-            },
-          ],
-        },
-      ],
+      type: "string",
+      key: "phone",
+      displayer: "Phone Number",
+      value: "1 800 222 000"
+    })
+    
+    this.addProp({
+      type: "string",
+      key: "text",
+      displayer: "Text",
+      value: "Started In"
+    })
+
+    this.addProp({
+      type: "string",
+      key:"year",
+      displayer: "Year",
+      value: "1995"
     });
+
     this.addProp({
-      type: "number",
-      key: "itemCount",
-      displayer: "Item count in a row",
-      value: 4,
+      type: "icon",
+      key:"phoneIcon",
+      displayer: "Icon",
+      value: "BsTelephoneOutbound"
     });
+
+    this.addProp({
+      type: "image",
+      key: "image1",
+      displayer: "image1",
+      value: "https://storage.googleapis.com/download/storage/v1/b/hq-composer-0b0f0/o/66a39f132f8a5b002ce6b70b?alt=media"
+    })
+
+    this.addProp({
+      type: "image",
+      key: "image2",
+      displayer: "image2",
+      value: "https://storage.googleapis.com/download/storage/v1/b/hq-composer-0b0f0/o/66a39f1b2f8a5b002ce6b716?alt=media"
+    })
+
+
+
+
   }
 
   getName(): string {
     return "Content 13";
   }
 
   render() {
+
     return (
       <div className={this.decorateCSS("container")}>
         <div className={this.decorateCSS("max-content")}>
-          <div className={this.decorateCSS("header")}>
-            <span className={this.decorateCSS("subtitle")}>{this.getPropValue("subtitle")}</span>
-            <h2 className={this.decorateCSS("title")}>{this.getPropValue("title")}</h2>
-          </div>
-          <div className={this.decorateCSS("card-child")}>
-            {this.castToObject<Card[]>("content-card").map(
-              
-              (card: Card, index: number) => (
-                <div className={this.decorateCSS("card-item-count")} style={{
-                  width: 90 / this.getPropValue("itemCount") + "%",
-                }}>
-                  <div key={`cnt-4-card-${index}`} className={this.decorateCSS("card")}>
-                    <i>
-                      <img alt="server" src={card.image} />
-                    </i>
-                    <h5 className={this.decorateCSS("card-title")}>{card.title}</h5>
-                    <p className={this.decorateCSS("card-description")}>{card.description}</p>
-                  </div>
+          <div className={this.decorateCSS("title-container")}>
+            <span className={this.decorateCSS("about-title")}>{this.getPropValue("aboutTitle")}</span>
```

Comment:
if prop is empty, do not render relevant elements. do this operation for all props